app/controllers/provider_interface/activity_log_controller.rb

```diff
@@ -15,12 +15,15 @@ def index
         end
       end
 
-      provider_ids = current_provider_user.providers.ids.join(', ')
-      if benchmark.real.between?(30, 115)
-        Rails.logger.info "Activity page slow for providers [#{provider_ids}] took #{benchmark.real.to_i} seconds"
-      end
-      if benchmark.real > 115
-        Rails.logger.info "Activity page timed out for providers [#{provider_ids}] took #{benchmark.real.to_i} seconds"
+      if benchmark.real > 30
```

Comment:
How do we feel about 30seconds? To me it feels like a very long time - I'd class a page as slow if it took over 5secs to load - 10sec also seems fair https://www.nngroup.com/articles/website-response-times/#toc-response-time-limits-2

Reply:
I'll put 5